src/html_components.py

```diff
@@ -15,202 +15,243 @@
 
 
 def get_overview_section(data, feature_list):
-    plot1 = html.Div([
-        html.Hr(),
-        dbc.Row([html.H3("Overview Section:")]),
-        dbc.Row(
-            [
-                dbc.Col(
-                    [
-                        dcc.Dropdown(
-                            id="q_selection",
-                            value="tech_org",
-                            options=[
-                                {"label": feature_list.loc[i]["variables3"], "value": i} for i in
-                                np.r_[data.columns[0:14], data.columns[15:18]]
-                            ],
-                        ),
-                    ],
-                    md=3,
-                ),
-                dbc.Col(
-                    [
-                        html.Iframe(
-                            id="gender_barplot",
-                            style={"width": "100%", "height": "500px", "border": "0px"},
-                        ),
-                    ]
-                ),
-            ]
-        )])
+    plot1 = html.Div(
+        [
+            html.Hr(),
+            dbc.Row(
+                dbc.Card(
+                    dbc.CardBody(
+                        [
+                            html.H4("Quick Info", className="card-title"),
+                            html.P(
+                                "The Mental Health in Tech Dashboard dashboard visualizes a dataset consisting of survey questions and responses about various aspects of the mental health of tech workers. It shows both an overview of the survey responses as well as additional visualizations specifically relevant to HR domain experts. The dataset used and github source are available in the links below.",
+                                className="card-text",
+                            ),
+                            dbc.CardLink(
+                                "Dataset",
+                                href="https://www.kaggle.com/osmi/mental-health-in-tech-2016",
```

Comment:
Can we also add `target="_blank"` for the link so that it opens in a new tab? In general, that's a better UX.
E.g.

`
dbc.CardLink("Dataset",
href="https://www.kaggle.com/osmi/mental-health-in-tech-2016",
target="_blank",
),
`